docs/core_components/pages/streamfield.rst

```diff
@@ -0,0 +1,412 @@
+Freeform page content using StreamField
+=======================================
+
+Wagtail's content model encourages thinking about the information on a page in a formal, structured way; for example, a page about an event is not just an assortment of headings, paragraphs and images, but has specific fields for dates, location, a list of speakers and so on. This ensures that the template developer has a known, predictable set of fields to work with, and can present that information consistently throughout the site in the most appropriate arrangement. It also allows the information to be re-used and repurposed, such as displaying an "events near you" listing.
```

Comment:
Happy to cut it down a bit. I wrote this part before I knew I'd be writing about streamfield for the torchbox.com blog, so I was probably leaping on my soapbox a bit too enthusiastically :-) Would probably make sense to have a short and to-the-point introduction here, and link to the blog post for further reading.